Cargo.toml

```diff
@@ -29,3 +30,6 @@ version-sync = "~0.7"
 # Some tests are rather slow. Furthermore, optimalisations tend to provoke races and UBs to
 # manifest, so we want to try that in tests if possible.
 opt-level = 1
+
+[features]
+parallel = ["rayon"]
```

Comment:
I think having it enabled as `rayon` is more appropriate. If someone wants to integrate with `FromParIter`, they already know they're using `rayon`. If there's any other parallel crate we might want to support, I think it should be possible to turn support on for either one separately.

Reply:
I thought I wrote this somewhere but you can't have a feature and a dependency with the same name, apparently.

Reply:
Actually, the trick is, every optional dependency becomes an *implicit* feature automatically. Therefore, you can't ask for it *explicitly* in the `[features]` section. But you still can depend on it by other features (as you just did), you can check for it in code (`#[cfg(feature = "rayon")]`) and you can depend on a crate with that feature turned on.

So, the solution here is actually to remove the `[features]` section directly and check for it in the code by the `#[cfg(feature = "rayon")]`.

Reply:
Ah, thanks. That worked.